### PR TITLE
gRPC example code and maven setup merged with master

### PR DESCRIPTION
**Pull request by Goju-Ryu**
This pull request is for merging the gRPC work with the rest of the proect. Unfortunately the branch was branched from master, not devellopment. This means that it cannot be merged directly with development but has to go through master first.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -81,11 +81,27 @@
                         <goals>
                             <goal>compile</goal>
                             <goal>compile-custom</goal>
-                            <goal>compile-python</goal>
                         </goals>
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```
Comment (GrobaxGrappleBlast) [inline pom.xml:96]: AWSOME DUDE

File: src/main/proto/proto_py.bat [hunk 2]
```diff
@@ -1 +1 @@
-START python -m grpc_tools.protoc -I --python_out=. --grpc_python_out=. ./grpc.proto
+START python -m grpc_tools.protoc -I --python_out=. --grpc_python_out=. ./grpc.proto
```
Comment (GrobaxGrappleBlast) [inline src/main/proto/proto_py.bat:1]: strange

Comment (GrobaxGrappleBlast) [reply src/main/proto/proto_py.bat:1]: the permissions have not been committed into the code, and must therefore be a settings or project structure thingy

File: .gitignore [hunk 3]
```diff
@@ -1,129 +0,0 @@
-# Byte-compiled / optimized / DLL files
```
Comment (GrobaxGrappleBlast) [inline .gitignore:1]: Why?

Comment (Goju-Ryu) [reply .gitignore:1]: Don't know what this is honestly. not something i have don manually.

File: src/main/proto/proto_py.bat (hunk 2 above)
Comment (Goju-Ryu) [reply src/main/proto/proto_py.bat:1]: Is it from the latest commit? how do you know it hasn't?

File: pom.xml (hunk 1 above)
Comment (Goju-Ryu) [reply pom.xml:96]: Haha, yeah ;)

**Comment by Goju-Ryu** [comment]
@GrobaxGrappleBlast you had a script to add to import python dependencies, right?

Also, if you want I can write the README about how to run the project. I don't think i have any problem with understanding your dependency script and i know most of the details about the requirements for the gRPC part of that explanation.

**Comment by GrobaxGrappleBlast** [comment]
ehm yes, ill get to that, im currently fiddeling elsewhere, but i will get to that.

**Pull request closed without merging**